libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState, MouseEvent, useRef } from 'react';
+
+import AppBar from '@material-ui/core/AppBar';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import Popper from '@material-ui/core/Popper';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined';
+import HelpOutlineOutlinedIcon from '@material-ui/icons/HelpOutlineOutlined';
+import MenuIcon from '@material-ui/icons/Menu';
+
+import { object } from '@storybook/addon-knobs';
+
+import './NavTop.scss';
+
+export interface INavTopProps {
+  title: string;
+  menuList: IMenuList[];
+  accountMenuList: IMenuList[];
+  logo: string;
+}
+export interface IMenuList {
+  title: string;
+}
+
+export interface IMenuSection {
+  menuList: IMenuList[];
+  open: boolean;
+  handleClose: any;
+  anchorEl: HTMLButtonElement;
+}
+
+const label = 'Props';
+const defaultValue = {
+  title: 'Page Name',
+  logo: 'https://kleeen.software/wp-content/uploads/2019/10/KS-Fit-Positive-Reg-White.png',
+  menuList: [
+    {
+      title: 'Dashboard',
+    },
+    {
+      title: 'Projects',
+    },
+    {
+      title: 'Teams',
+    },
+    {
+      title: 'Expenses',
+    },
+    {
+      title: 'Departments',
+    },
+  ],
+  accountMenuList: [
+    {
+      title: 'User Preferences',
+    },
+    {
+      title: 'Logout',
+    },
+  ],
+};
+
+const MenuListSection = (props: IMenuSection) => {
+  return (
+    <ClickAwayListener onClickAway={props.handleClose}>
+      <Popper open={props.open} anchorEl={props.anchorEl} role={undefined} transition disablePortal>
+        {({ TransitionProps, placement }) => (
+          <Grow
+            {...TransitionProps}
+            style={{ transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom' }}
+          >
+      <Paper>
+        <MenuList>
+          {props.menuList.map(({ title }) => <MenuItem>{title}</MenuItem>)}
+        </MenuList>
+      </Paper>
+          </Grow>
+        )}
+      </Popper>
+    </ClickAwayListener>
+  );
+};
+
+export const NavTop = (xprops: INavTopProps) => {
+  const props = object(label, defaultValue);
+
+  const [openAccount, setOpenAccount] = useState(false);
+  const [openMainMenu, setOpenMainMenu] = useState(false);
+
+  const mainMenu = useRef<HTMLButtonElement>(null);
+  const accountMenu = useRef<HTMLButtonElement>(null);
+
+  const handleToggleAccount = () => {
+    setOpenAccount(prevOpenAccount => !prevOpenAccount);
```

Comment:
```suggestion
    setOpenAccount(!prevOpenAccount);
```

I think this achieves the same result 🤔